File: fortran/fizzbuzz.f90
program main
  integer :: i
  do i = 100, 1, -1
     if (mod(i,3) == 0 .and. mod(i,5) == 0) then
        write (*,'(A8)') 'fizzbuzz'
     elseif (mod(i,3) == 0) then
        write (*,'(A4)') 'fizz'
     elseif (mod(i,5) == 0) then
        write (*,'(A4)') 'buzz'
     else
        write (*,'(I0)') i
     end if
  end do
end program main

File: c/FizzBuzz.c
#include <stdio.h>

int main(void)
{
  for (int i = 100; i > 0; i--) {
    if (i%3 == 0 && i%5 == 0)
      printf("%d - FizzBuzz\n", i);
    else if (i%3 == 0)
      printf("%d - Fizz\n", i);
    else if (i%5 == 0)
      printf("%d - Buzz\n", i);
    else
      printf("%d - %d\n", i, i);
  }
  
  return 0;
}
